spinn_front_end_common/interface/interface_functions/dsg_region_reloader.py

```diff
@@ -86,6 +81,20 @@ def __call__(
         for vertex in application_vertices_to_reset:
             vertex.mark_regions_reloaded()
 
+        # FIXME: Needs correct logic for whether to delete
+        if self._DELETE_TEMP_DIRS:
```

Comment:
This is the line that needs to be something else, but I don't know what.

Reply:
having reviewed, i also think a cfg param so that we can change it without going into code.